ext/pdo/php_pdo.h

```diff
@@ -49,17 +49,18 @@ extern zend_module_entry pdo_module_entry;
 
 PHP_MINIT_FUNCTION(pdo);
 PHP_MSHUTDOWN_FUNCTION(pdo);
+PHP_RINIT_FUNCTION(pdo);
+PHP_RSHUTDOWN_FUNCTION(pdo);
 PHP_MINFO_FUNCTION(pdo);
 
 ZEND_BEGIN_MODULE_GLOBALS(pdo)
 	zend_long  global_value;
+	HashTable  *pdo_pdbh_to_delete;
```

Comment:
There was an announce from Davey http://news.php.net/php.internals/94330 . But i don't think it was just ok with the previous versions as well, AFAIR (and how i handled it) the practice is always to freeze both API/ABI, like fe https://fedoraproject.org/wiki/Updates_Policy#Beta_to_Pre_Release and other projects do. But there's always room for an exception, sometimes a breach is acceptable or even unavoidable, so @dshafik and @krakjoe would be in charge for that.

Thanks.